democracy_now_scraper/extract_dn.py

```diff
@@ -0,0 +1,264 @@
+"""A file to extract information from various Fox News RSS feeds."""
+
+from datetime import datetime, timedelta
+
+import grequests
+from bs4 import BeautifulSoup
+
+
+def fetch_response_html(response) -> BeautifulSoup:
+    """Return html content of response"""
+
+    return BeautifulSoup(response.content, "html.parser")
+
+
+def chunk_links(links: list[str], chunk_size: int) -> list[list[str]]:
+    """
+    Split the list of links into smaller chunks.
+    """
```

Comment:
Small thing, but might want to standardise the formatting of the one line docstrings.

Reply:
Good spot